Source/Core/InputCommon/ControllerEmu/ControllerEmu.h

```diff
@@ -32,7 +32,10 @@ class EmulatedController
 
   virtual void LoadConfig(IniFile::Section* sec, const std::string& base = "");
   virtual void SaveConfig(IniFile::Section* sec, const std::string& base = "");
-  void UpdateDefaultDevice();
+
+  const ciface::Core::DeviceQualifier& GetDefaultDevice();
```

Comment:
Shouldn't the member function be const qualified as well?

Reply:
Yeah, I missed that.